packages/app-store/salesforce/lib/CrmService.ts

```diff
@@ -1336,22 +1332,22 @@ export default class SalesforceCRMService implements CRM {
       `SELECT Id, Email FROM ${SalesforceRecordEnum.CONTACT} WHERE Email = '${email}' LIMIT 1`
     );
 
-    if (contactsQuery.records.length) {
+    if (contactsQuery.records.length > 0) {
```

Comment:
Instead of an if statement this is the ideal place to extract into a sub function :)